provider/internal/java/dependency.go

```diff
@@ -411,7 +411,7 @@ func (p *javaServiceClient) parseDepString(dep, localRepoPath, pomPath string) (
 	fp := resolveDepFilepath(&d, p, group, artifact, localRepoPath)
 
 	d.Labels = addDepLabels(p.depToLabels, d.Name)
-	d.FileURIPrefix = fmt.Sprintf("file://%v", filepath.Dir(fp))
+	d.FileURIPrefix = fmt.Sprintf("file://%v", filepath.Dir(strings.TrimPrefix(fp, localRepoPath)))
```

Comment:
I don't think we should be doing this,

Reply:
why?

Reply:
this will break the dependency filtering logic, either we fix it in both places or don't change the fileURIPrefix at all, because this is mainly just an internal field

Reply:
@eemcmullan this could be causing the failures too then maybe?

Reply:
Think so. Probably best then to trim this prefix in https://github.com/konveyor/analyzer-lsp/blob/main/cmd/analyzer/main.go#L423